### PR TITLE
Readme update to reflect updated wiki

### PR DESCRIPTION
**Pull request by lfportal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,15 +117,16 @@ npx @airtasker/spot generate --contract api.ts
...
-* [`spot checksum SPOT_CONTRACT`](#spot-checksum-spot_contract)
-* [`spot docs SPOT_CONTRACT`](#spot-docs-spot_contract)
-* [`spot generate`](#spot-generate)
-* [`spot help [COMMAND]`](#spot-help-command)
-* [`spot init`](#spot-init)
-* [`spot lint SPOT_CONTRACT`](#spot-lint-spot_contract)
-* [`spot mock SPOT_CONTRACT`](#spot-mock-spot_contract)
-* [`spot validate SPOT_CONTRACT`](#spot-validate-spot_contract)
-* [`spot validation-server SPOT_CONTRACT`](#spot-validation-server-spot_contract)
+
```
Comment (committedteadrinker) [inline README.md:120]: just confirming, is this just an extra space?

File: README.md [hunk 2]
```diff
@@ -87,11 +93,12 @@ We built Spot with this in mind. Instead of having to juggle various API format
 [![License](https://img.shields.io/npm/l/@airtasker/spot.svg)](https://github.com/airtasker/spot/blob/master/package.json)
 
 <!-- toc -->
-* [Spot](#spot)
-* [Why we built Spot](#why-we-built-spot)
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
+
```
Comment (committedteadrinker) [inline README.md:96]: Is this also just an extra space? Just nit

**Review by committedteadrinker — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lfportal) [reply README.md:120]: yup, this stuff gets butchered during the release

File: README.md (hunk 2 above)
Comment (lfportal) [reply README.md:96]: yup, this stuff gets butchered during the release

**Pull request merged**